docs/antrea-network-policy.md

```diff
@@ -426,14 +426,18 @@ to a separate file (`/var/log/antrea/networkpolicy/np.log`) on the Node on
 which the rule is applied. These log files can then be retrieved for further
 analysis. By default, rules are not logged. The example policy logs all
 traffic that matches the "DropToThirdParty" egress rule, while the rule
-"AllowFromFrontend" is not logged. The rules are logged in the following
-format:
+"AllowFromFrontend" is not logged. Specifically for drop and reject rules,
+deduplication is applied to simplify the logs. By default, the buffer length is 1 second.
```

Comment:
this makes it sound like the buffer length is configurable and default value is 1s.